pkg/cmd/server/origin/handlers.go

```diff
@@ -21,30 +21,32 @@ import (
 	"github.com/openshift/origin/pkg/authorization/authorizer"
 )
 
+// TODO once we have a MuxHelper we will not need to hardcode this list of paths
+// RootPaths are paths we server on from root
+var RootPaths kapi.RootPaths = kapi.RootPaths{Paths: []string{
+	"/api",
+	"/api/v1beta3",
```

Comment:
It was only for consistency with upstream.  I am utterly ambivalent.

On Mon, Aug 10, 2015 at 4:48 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In pkg/cmd/server/origin/handlers.go
> <https://github.com/openshift/origin/pull/4074#discussion_r36682438>:
>
> > @@ -21,30 +21,32 @@ import (
> >  	"github.com/openshift/origin/pkg/authorization/authorizer"
> >  )
> >
> > +// TODO once we have a MuxHelper we will not need to hardcode this list of paths
> > +// RootPaths are paths we server on from root
> > +var RootPaths kapi.RootPaths = kapi.RootPaths{Paths: []string{
> > +	"/api",
> > +	"/api/v1beta3",
>
> @smarterclayton <https://github.com/smarterclayton> do we want to expose
> the API versions at the root or require traversing to /api to get the list
> of enabled versions? This static list won't reflect which versions are
> actually enabled/disabled (which we make configurable)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4074/files#r36682438>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift